numpy/core/src/multiarray/compiled_base.c

```diff
@@ -1690,11 +1707,15 @@ pack_bits(PyObject *input, int axis)
     return NULL;
 }
 
+typedef char lookup[256][8];
+
 static PyObject *
-unpack_bits(PyObject *input, int axis, PyObject *count_obj)
+unpack_bits(PyObject *input, int axis, PyObject *count_obj, char order)
 {
     static int unpack_init = 0;
-    static char unpack_lookup[256][8];
+    static lookup unpack_lookup_b;
+    static lookup unpack_lookup_l;
+    lookup *unpack_lookup;
```

Comment:
nit: Should be `lookup const *unpack_lookup`

Reply:
added const

Reply:
actually removed this all